### ISSUE TITLE
native_constant_invocation applies to catch section without variables

### ISSUE THREAD

**Issue by BoShurik** [FriendsOfPHP/PHP-CS-Fixer#5684]
## Bug report

PHP 8.0.2
PHP CS Fixer 3.0.0

### Code snippet that reproduces the problem

```php
return (new PhpCsFixer\Config())
    ->setFinder($finder)
    ->setRules([
        'native_constant_invocation' => true,
    ])
;
```

```php
try {
    $date = new \DateTime();
} catch (\Exception) {
}
```
Suggests
```php
try {
    $date = new \DateTime();
} catch (Exception) {
}
```

**Comment by kubawerlos** [comment]
@BoShurik can you take a look at the PR ☝🏼 (and maybe drop a review)?

**Comment by BoShurik** [comment]
@kubawerlos works for me

**Comment by niji-mneute** [comment]
The bug is still present in version 3.5.0

```php
try {
    // ...
} catch (\InvalidArgumentException|\LogicException) {
    // ...
}
```

Suggets : 
```php
try {
    // ...
} catch (\InvalidArgumentException|LogicException) {
    // ...
}
```

**Comment by SpacePossum** [comment]
thanks for reporting @niji-mneute  , proposed fix is here https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6264

**Comment by benjaminmal** [comment]
I can confirm that #6264 fix #6271. Thank you @SpacePossum!
